app/request.py

```diff
@@ -57,6 +57,8 @@ def send_tor_signal(signal: Signal) -> bool:
                     for line in conf:
                         pass
                     secret = line
+                    if '\n' in line:
+                        secret = line.strip('\n')
```

Comment:
Could be simplified to:
```suggestion
                    secret = line.strip('\n')
```